docs/howto/platform-sdk-quickstart-guide.md

```diff
@@ -0,0 +1,194 @@
+    create a custom tool or plugin:\
+    \
+    `ayx_plugin_cli create-ayx-plugin`\
+    \
+    The command asks you to provide information about your tool or
+    plugin through a series of prompts. Note that this operation can
+    take a minute to complete.
+
+Once you have stepped through the above command, you can go to the
+location of your workspace and view the files that were created for the
+new plugin.
+
+For more information about the CLI, please visit the [reference
+documents](https://help.alteryx.com/developer-help/ayx-plugin-cli-overview).
+
+### UI SDK
+
+The UI SDK consists of 2 pieces.
+
+-   The 1st piece is a React-based [component
+    library](https://alteryx.github.io/alteryx-ui/)
+    that enables you to build the interface for your tool or plugin.
+-   The 2nd piece is a [communication
+    bridge](https://alteryx.github.io/react-comms/)
+    that lets you interact with Alteryx Designer and persists the values
+    that you want to save between runs of a workflow, clicking on and
+    off a tool. These are also the values that are available to the
+    engine (back end) of your tool.
+
+For the real-time Dev Harness that assists in tool UI development, refer to [Dev Harness.](https://github.com/alteryx/dev-harness)
+
+Follow these steps to get started with the UI SDK:
+
+1.  To start on the user interface of your project, navigate to the
+    folder that was created for your workspace in the last step of the
+    CLI section above. In this workspace folder, there is a UI directory
+    for the workspace. There is a folder inside of this directory with
+    the same name as the plugin you created.
+2.  Navigate to [\[workspace directory\]\\ui\\\[plugin
+    name\]\\src\\src]{.path} for the tool you are working on.
+3.  Open the index.tsx file. This is the primary
+    [React](https://reactjs.org/)
+    file for the tool UI.
+4.  Make a change to the file. For example, edit the text of a tag, or
+    add a [new
+    component](https://alteryx.github.io/alteryx-ui/)
+    or HTML element to the page.
+5.  Save the file.
+6.  For a quick preview of your tool in a browser, run this command:\
+    \
+    `npm run start`
+7.  To view your tool in Alteryx Designer, run this command from the
+    root workspace directory. This command builds and installs your
+    plugin into Designer.\
+    \
+    `ayx_plugin_cli designer-install`
+
+Open Alteryx Designer, search for your tool, place it on the canvas, and
+view the changes.
+
+For more information about the UI SDK, please visit the [reference
+documents](https://alteryx.github.io/alteryx-ui/).
+
+
+### Python SDK
+
+#### Warning
+V1 of AYX Python SDK is only compatible with Alteryx Designer versions
+2021.2 and 2021.3. For the best experience, and to ensure compatibility
+with the most recent versions of Designer, please use [version 2 of AYX
+Python
+SDK](https://help.alteryx.com/developer-help/ayx-python-sdk-v2).
```

Comment:
Point this to the new reference on ayx-developer-sdk repo.

This should point to the corresponding markdown version